client/src/styles/global.scss

```diff
@@ -329,3 +331,89 @@ $primary-color: #112f4e;
     }
   }
 }
+
+
+/* the > is for child only syntax in css. This is required so that lists that are nested under the
+      list are not affected (e.g. <ul><li><p><ul><li>not styled</li></ul></p></li><ul>*/
+.j40-process-list-wrapper > {
+  $j40-steps-list-color: #002D3F;
+  ul {
+    margin-top: 3rem;
+    margin-bottom: 1rem;
+    padding-inline-start: 0;
+
+    > li {
+      list-style-type: none;
+      position: relative;
+      margin-left: 2.5rem;
+      /* do the lines */
+      border-left: .2rem solid $j40-steps-list-color;
+      padding-left: 1.88rem;
+      /* removes gap between line and circle */
+      margin-top: -1rem;
+      padding-bottom: 2rem;
+    }
+
+    > li:before {
+      content: " ";
+      border: 1rem solid $j40-steps-list-color;
+      border-radius: 100rem;
+      height: 0; /* it's all controlled by the border */
+      width: 0;
+      margin-top: -0.5rem;
+      margin-left: -3rem;
+      position: absolute;
+    }
+
+    > li:last-child {
+      /* no line on last item */
+      border-left: .2rem solid transparent;
+      margin-top: -1.9rem;
+    }
+  }
+}
+
+/* The math equation on the methodology page.. which is a royal pain */
+.j40-math-division-container {
```

Comment:
Looks great!

It seems we have two ways to style in this project:

1. Place styles of the component in the global sass file
2. Place styles for the component in the component.module.scss file

How should we determine when to use which?

I haven't been using global sass file too much but I am starting to. I am using global sass file for styles that exist on more than one page (for example the h1 styling for each page) and need to shared across thee app. I have been using components module styling for all component based styling.

Reply:
Yeah. I started to go down this path, but the scss is so complicated as it is that I was saving abstractions like localizations and componentizing/typescript names (for scss) for a second pass. Probably a good idea to generate a bug so these tasks don't get lost.

My personal opinion, but I'm still on the fence about turning one file `scoreStepsList.jsx`

Reply:
Might be worthwhile to discuss the complexity you mention here at some point. I'm thinking that most of the component specific scss written in the `global` file could theoretically (might be wrong here) be placed in a module no?

Reply:
Yeah. It's a tradeoff between the complexity of having two additional files and extra layers of indirection in the project for scoping versus using the scoping built into scss and css. It probably has more to do with me being more comfortable with scss than with generating scss typescript interfaces to replace classname strings.

I'm happy to convert it over, I'd just prefer to make sure what we have is OK with design because, when there's extra levels of indirection for styles, I find big edits are trickier. (react -> typescript interface names -> scss vs react -> scss).

Reply:
agreed